packages/react/types/tests.tsx

```diff
@@ -6,7 +6,9 @@ import {
   css,
   jsx,
   keyframes,
-  withEmotionCache
+  withEmotionCache,
+  Interpolation,
+  Theme
```

Comment:
Oops, sorry. I forgot to delete unused imports. I’ll add additional commit as soon as possible.